File: lua/refactoring/tests/refactor/106/cpp/extract-block/extract_block.expected.cpp
#include <iostream>

void foo_bar(int a) {
  int test = 1, test_other = 1;

  for (int idx = test - 1; idx < test_other; idx++) {
    std::cout << idx << " " << a << std::endl;
  }
}


void simple_function(int a) {
  foo_bar(a);
}

File: lua/refactoring/tests/refactor/106/js/extract-block/extract_block.expected.js

function foo_bar(a) {
    let test = 1;
    let test_other = 11
    for (let idx = test - 1; idx < test_other; ++idx) {
        console.log(idx, a)
    }
}

function simple_function(a) {
    foo_bar(a);
}
